packages/blade/src/components/Table/types.ts

```diff
@@ -363,8 +371,9 @@ type TablePaginationCommonProps = {
    * The default page size.
    * Page size controls how rows are shown per page.
    * @default 10
+   * consider using virtualization for large page sizes
    **/
-  defaultPageSize?: 10 | 25 | 50;
+  defaultPageSize?: 10 | 25 | 50 | 100 | 200;
```

Comment:
we should ignore this prop in virtualization I feel. People might want to render more than 200 items also

Reply:
will revert this change back to   ``` defaultPageSize?: 10 | 25 | 50 ```

can't think of any use case where people will use pagination with virtualization